.github/workflows/main.yml

```diff
@@ -154,7 +235,7 @@ jobs:
             USER_1_EMAIL=$USER_1_EMAIL \
             --dart-define \
             USER_1_PASSWORD=$USER_1_PASSWORD \
-            -d $SIMULATOR_UDID
+            --dart-define
 
```

Comment:
Why the empty `--dart-define`?

Reply:
And why do you change this (inside iOS action) in this PR?

Reply:
was a mistake